tests/testthat/_snaps/bundlePackage.md

```diff
@@ -0,0 +1,11 @@
+# errors if dependencies aren't installed
+
+    Code
+      bundlePackages(app_dir, appMode = "rmd-static", assetTypeName = "asset")
+    Warning <simpleWarning>
+      Package 'doesntexist1' not available in repository or locally
+      Package 'doesntexist2' not available in repository or locally
+    Error <simpleError>
+      Unable to retrieve package records for the following packages:
+      - 'doesntexist1', 'doesntexist2'
```

Comment:
It looks as if the Windows test failures are due to differences in output quoting.

This output comes from Packrat and is sensitive to `shQuote`:

https://github.com/rstudio/packrat/blob/9917985132f2dce44ed1ef9d3a5aa7679a7c411c/R/pkg.R#L259-L263

Reply:
Ok, that's an easy fix.